flocker/node/agents/functional/test_ebs.py

```diff
@@ -182,6 +188,132 @@ def test_next_device_in_use(self):
         self.assertEqual(result, u"/dev/sdg")
 
+    def test_create_volume_gold_profile(self):
+        """
+        Requesting ``gold`` profile during volume creation honors
+        ``gold`` attributes.
+        """
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.GOLD)
+
+    @capture_logging(assertHasMessage, CREATE_VOLUME_FAILURE)
+    def test_create_too_small_volume_gold_profile(self, logger):
+        """
+        Too small volume for ``gold`` attributes creates volume with
+        ``default`` profile.
+        """
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.GOLD, created_profile=MandatoryProfiles.DEFAULT,
+            size_GiB=1)
+
+    def test_create_largest_volume_gold_profile(self):
+        """
+        Largest volume with ``gold`` profile.
+        """
+        raise SkipTest(u'This test is expensive (money).')
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.GOLD, size_GiB=16*1024)
+
+    def test_create_too_large_volume_gold_profile(self):
+        """
+        Too large volume (> 16TiB) for ``gold`` profile.
+        """
+        self.assertRaises(EC2ResponseError,
+                          self._assert_create_volume_with_mandatory_profile,
+                          MandatoryProfiles.GOLD,
+                          size_GiB=1024*1024)
+
+    def test_create_volume_silver_profile(self):
+        """
+        Requesting ``silver`` profile during volume creation honors
+        ``silver`` attributes.
+        """
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.SILVER)
+
+    def test_create_too_large_volume_silver_profile(self):
+        """
+        Too large volume (> 16TiB) for ``silver`` profile.
+        """
+        self.assertRaises(EC2ResponseError,
+                          self._assert_create_volume_with_mandatory_profile,
+                          MandatoryProfiles.SILVER,
+                          size_GiB=1024*1024)
+
+    def test_create_largest_volume_silver_profile(self):
+        """
+        Largest volume with ``silver`` profile.
+        """
+        raise SkipTest(u'This test is expensive (money).')
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.SILVER, size_GiB=16*1024)
+
+    def test_create_volume_bronze_profile(self):
+        """
+        Requesting ``bronze`` profile during volume creation honors
+        ``bronze`` attributes.
+        """
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.BRONZE)
+
+    def test_create_too_large_volume_bronze_profile(self):
+        """
+        Too large volume for ``bronze`` profile.
+        """
+        self.assertRaises(EC2ResponseError,
+                          self._assert_create_volume_with_mandatory_profile,
+                          MandatoryProfiles.SILVER,
```

Comment:
That's silver, not bronze. And again not sure this test tells us anything useful.